### ISSUE TITLE
[request] returning 500 error should contain an "error" field

### ISSUE THREAD

**Issue by n3wscott** [pmorie/osb-broker-lib#36]
the spec says this: https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#service-broker-errors

It would be nice to be able to return/define that error or have the lib default to "InternalServerError" or something like this...

**Comment by n3wscott** [comment]
I could see closing this as not important.

**Comment by LiliC** [comment]
No, I think it's valid, just not the biggest priority probably right now. Lets leave it open! :)
